src/context/Router/index.tsx

```diff
@@ -169,7 +170,9 @@ const router = createBrowserRouter(
                         path={authenticatedRoutes.home.path}
                         element={
                             <Suspense fallback={null}>
-                                <HomePage />
+                                <DashboardWelcomeProvider>
```

Comment:
Since the hook `useShowDashboardWelcome` I _think_ we're good to remove this to make sure the surface area of possible issues is as small as possible.